mmdet3d/apis/inference.py

```diff
@@ -177,7 +177,9 @@ def inference_multi_modality_detector(model: nn.Module,
                                       imgs: Union[str, Sequence[str]],
                                       ann_file: Union[str, Sequence[str]],
                                       cam_type: str = 'CAM_FRONT'):
```

Comment:
If we only support kitty dataset for multi-modality detector, the default cam_type should be `CAM2'?